golang/cosmos/x/swingset/types/msgs_test.go

```diff
@@ -151,28 +150,9 @@ func TestInstallBundle_ValidateBasic(t *testing.T) {
 			msg: &MsgInstallBundle{
 				Submitter:        addr,
 				CompressedBundle: []byte{1, 2, 3},
-				UncompressedSize: bundleUncompressedSizeLimit - 1,
+				UncompressedSize: int64(DefaultBundleUncompressedSizeLimitBytes - 1),
 			},
 		},
-
-		{
-			name: "limit",
-			msg: &MsgInstallBundle{
-				Submitter:        addr,
-				CompressedBundle: []byte{1, 2, 3},
-				UncompressedSize: bundleUncompressedSizeLimit,
-			},
-			shouldErr: true,
-		},
-		{
-			name: "max",
-			msg: &MsgInstallBundle{
-				Submitter:        addr,
-				CompressedBundle: []byte{1, 2, 3},
-				UncompressedSize: math.MaxInt64,
-			},
-			shouldErr: true,
-		},
```

Comment:
Why did these have to be removed?

Reply:
`ValidateBasic` no longer has enough information on hand to enforce the chunk size limit, so the enforcement had to move to the keeper. There are new `validate*` methods on the message server and the new tests are largely to compensate for the loss of these tests.